src/Components/HomeSection/HomeSection.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './HomeSection.css'
+import Card from '../../Components/Card/Card'
+
+function HomeSection({title, data}) {
+    return (
+        <div>
+            <h1 className="section-title">{title}</h1>
+            <div className="grid-box">
+                {
+                    [0, 1, 2, 3].map(item => {
```

Comment:
What does this section do?

Reply:
Got left unchecked, during code refactoring. Will Delete this.